linelist_investigations/report_sources/aaa_clean_linelist_2019-10-18.Rmd

```diff
@@ -373,8 +373,14 @@ This section defines age classes - see code for details.
 ```{r clean_age}
 
 x <- x %>%
-  mutate(
-      age = as.numeric(gsub("_", ".", age)),
+  mutate(age = gsub("_", ".", age))
```

Comment:
This is dangerous, as it will be converting age to `character` if it is a number, e.g.:
```r
> gsub("_", ".", 1.23)
[1] "1.23"
```
Best condition the whole thing on `age` being a `character` using `if (is.character(x$age)) {...}`